ToDoApp/ToDoApp/ToDoListView.swift

```diff
@@ -0,0 +1,99 @@
+                .init(title: "Cooking", deadline: Date(), status: true),
+                .init(title: "Cooking", deadline: Date(), status: true),
+                .init(title: "Cooking", deadline: Date(), status: false)
+                
+            ]),
+            .init(title: "All To-Do List", toDoItems: [
+                .init(title: "Laundry", deadline: Date(), status: false),
+                .init(title: "Laundry", deadline: Date(), status: true),
+                .init(title: "Laundry", deadline: Date(), status: false),
+                .init(title: "Laundry", deadline: Date(), status: true),
+                .init(title: "Laundry", deadline: Date(), status: false),
+                .init(title: "Laundry", deadline: Date(), status: false),
+                .init(title: "Laundry", deadline: Date(), status: false),
+                .init(title: "Laundry", deadline: Date(), status: false),
+                .init(title: "Painting", deadline: Date(), status: true)
+            ])
+        ]
+    var body: some View {
+        NavigationStack{
+            List {
+                ForEach(ToDoListView.items) { tasks in
+                    HStack(spacing: 6) {
+                        Text(tasks.title)
+                            .foregroundColor(Color(.caption))
+                            .font(.system(size: 12))
+                            .fontWeight(.medium)
+                        Text("\(tasks.toDoItems.capacity) tasks left")
+                            .foregroundColor(.gray)
+                            .font(.system(size: 12))
+                            .fontWeight(.thin)
+                    }
+                    ForEach(tasks.toDoItems) { todo in
+                        CellTask(todo: todo)
+                    }
+                }
+                Spacer()
+            }
+            .navigationTitle("To-Do List")
+            .listStyle(.plain)
+            .padding(.leading, 16)
+        }
+    }
+}
+struct CellTask: View {
+    var todo: TaskModel
+    var body: some View {
+        VStack(alignment: .leading) {
+            HStack(spacing: 30) {
+                Button( action: {
+                    print("Change status")
+                }, label: {
+                        Image(systemName: (todo.status ? "checkmark.circle" : "circle"))
+                            .frame(minHeight: 22)
+                            .clipShape(Circle())
+                            .foregroundColor(.blue)
+                    }
+                )
+                VStack(alignment: .leading) {
+                    Text(todo.title)
+                        .bold()
+                        .font(.system(size: 17))
+                    Text(formatDate(deadline: todo.deadline))
+                        .font(.system(size: 15))
+                }
+            }
+        }
+    }
+    func formatDate(deadline: Date) -> String {
+        return deadline.formatted(.iso8601.month().day().year().dateSeparator(.dash))
```

Comment:
this date formatted is completely different from Figma. ISO8601 date format mostly used for computers (requests/response strings etc). you can simply use .formatted() and that would be enough. even though according to Figma the time should not be displayed this is a mistake. since the user can specify deadline time, the user should also be able to see the specified time

Reply:
I'm still struggling to format it in the exact format like "Dec 8th, 2022," but I'm just able to get it to look like "Dec 8, 2022"

Reply:
Sometimes you should not sticky follow the design, and here is why: there are a lot of different date formats in the world. For example in USA it is MM/dd/yyyy, in Canada it is yyyy-MM-dd, in some European countries it is dd.MM.yyyy and so on. So to avoid confusion for the users the date format should be localized. That's why it is almost never a good idea to hardcore a date format. iOS ecosystem provides us with a banch of predefined date and time styles which represents long, medium, short and full style. Which will work for most of the cases. In our case Figma specifies that only date should be shown. But again, this is a mistake, since the user apart of a date can set a time as a deadline. That's why the time should be also displayed to the user. So, method `formatted()` on the Date type fully fulfills or needs